### PR TITLE
Add hardware requirements doc

### PR DESCRIPTION
**Pull request by dincho**

### CODE REVIEW

File: docs/hardware-requirements.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+So a reasonable disk space allocation for 1 year horizon would be **450GB**.
+
+### Lightweight Node
+
+Database size as of the moment of this writing is 60GB and grows with 3GB per month.
+So a reasonable disk space allocation for 1 year horizon would be **100GB**.
+
+## Bandwidth
+
+During the normal operations the node needs up to 30KB/s inbound and up to 30KB/s outbound traffic.
```
Comment (hanssv) [inline docs/hardware-requirements.md:46]: With "normal" being one transaction every 10 seconds? I would expect a fully utilised chain to be more like 300-500 KB/s for gossip.

**Review by hanssv — DISMISSED** [review]

File: docs/hardware-requirements.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Introduction
+
+Hardware requirements may vary based in what mode the node is running, the network and it is supposed to run full sync or restore from database snapshot. Running a full node to do a full sync being the most resource intensive.
+Requirements would also vary based on the node configuration, API requests and eventually on future consensus protocol and features.
+
+Last update: **October 1, 2024**
+
+## Summary
+
+Running a full node with full since now and then (i.e. once per month):
```
Comment (happi) [inline docs/hardware-requirements.md:10]: should "since" be "sync"?

**Review by happi — APPROVED** [review]

File: docs/hardware-requirements.md (hunk 1 above)
Comment (dincho) [reply docs/hardware-requirements.md:46]: This is measured on current mainnet. I'll add your numbers as well

Comment (dincho) [reply docs/hardware-requirements.md:48]: 4b613988c

File: docs/hardware-requirements.md (hunk 2 above)
Comment (dincho) [reply docs/hardware-requirements.md]: 4b613988c

**Review by happi — APPROVED** [review]

**Review by hanssv — APPROVED** [review]

**Pull request merged**